frontend/src/components/3.Preferences/AutocompleteSearchBox.js

```diff
@@ -59,8 +59,7 @@ const AutocompleteSearchBox = ({ setLocation, sendCoordinates }) => {
                   <div
                     {...getSuggestionItemProps(suggestion)}
                     key={index}
-                    className={`SuggestionItem ${classes}`}
-                  >
+                    className={`SuggestionItem ${classes}`}>
```

Comment:
Its like that because its an easy way to add multiple classes. ${classes} is a conditional class applied.

Preferences.js uses that notation because it uses a module.css file. I don't think its appropriate personally to require this change in this PR but if you want, then we could make a new issue request to change the way CSS files are used in this app for consistency.

Reply:
good spotting! I agree. Is prettier doing this automatically or did you change it Sree?

Reply:
Yes I agree, I've fixed this now